solutions/30/01.py

```
@@ -0,0 +1,43 @@
+class Solution:
+    def findSubstring(self, s: str, words: List[str]) -> List[int]:
```

Ruff (0.14.7)

Undefined name `List`

(F821)